src/main/kotlin/com/team4099/robot2023/config/constants/FieldConstants.kt

```diff
@@ -0,0 +1,265 @@
+package com.team4099.robot2023.config.constants
+
+import edu.wpi.first.wpilibj.DriverStation
+import edu.wpi.first.wpilibj.DriverStation.Alliance
+import org.team4099.lib.geometry.Pose2d
+import org.team4099.lib.geometry.Pose3d
+import org.team4099.lib.geometry.Rotation3d
+import org.team4099.lib.geometry.Translation2d
+import org.team4099.lib.geometry.Translation3d
+import org.team4099.lib.units.base.feet
+import org.team4099.lib.units.base.inches
+import org.team4099.lib.units.derived.Angle
+import org.team4099.lib.units.derived.cos
+import org.team4099.lib.units.derived.radians
+import org.team4099.lib.units.derived.sin
+
+/**
+ * Contains various field dimensions and useful reference points. Dimensions are in meters, and sets
+ * of corners start in the lower left moving clockwise.
+ *
+ * All translations and poses are stored with the origin at the rightmost point on the BLUE ALLIANCE
+ * wall. Use the [.allianceFlip] and [.allianceFlip] methods to flip these values based on the
+ * current alliance color.
+ */
+object FieldConstants {
```

Comment:
Same as above -- this should be in a separate PR

Reply:
i think this was also rebased at some point, idk why it shows up as an addition